<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessDetailView</class>
 <widget class="QTabWidget" name="ProcessDetailView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tab_image">
   <attribute name="title">
    <string>Image</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>PID</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="lineEdit_pid">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Parent PID</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="lineEdit_ppid">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Session ID</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLineEdit" name="lineEdit_tpgid">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLineEdit" name="lineEdit_tty">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="lineEdit_sid">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Controlling Terminal</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Foreground Process Group ID</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="lineEdit_pgid"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Start Date</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Process Group ID</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QDateTimeEdit" name="dateTimeEdit_starttime">
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="accelerated">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>State</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QLabel" name="label_state">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>GroupBox 3</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTableWidget" name="tableWidget_uids">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="rowCount">
          <number>2</number>
         </property>
         <property name="columnCount">
          <number>4</number>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>75</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>50</number>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>Uid</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Gid</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Real</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Effective</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Saved set</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Filesystem</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2" rowspan="2">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Flags</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_pf_exiting">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PF_EXITING</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pf_exitpidone">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PF_EXITPIDONE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pf_vcpu">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PF_VCPU</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pf_wq_worker">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PF_WQ_WORKER</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pf_forknoexec">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PF_FORKNOEXEC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pf_mce_process">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PF_MCE_PROCESS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pf_superpriv">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PF_SUPERPRIV</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pf_dumpcore">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PF_DUMPCORE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pf_signaled">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PF_SIGNALED</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pf_memalloc">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PF_MEMALLOC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pf_nproc_exceeded">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PF_NPROC_EXCEEDED</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pf_used_math">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PF_USED_MATH</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pf_used_async">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PF_USED_ASYNC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pf_nofreeze">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PF_NOFREEZE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pf_frozen">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PF_FROZEN</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pf_fstrans">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PF_FSTRANS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pf_kswapd">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PF_KSWAPD</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pf_memalloc_noio">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PF_MEMALLOC_NOIO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pf_less_throttle">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PF_LESS_THROTTLE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pf_kthread">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PF_KTHREAD</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pf_randomize">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PF_RANDOMIZE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pf_swapwrite">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PF_SWAPWRITE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pf_no_setaffinity">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PF_NO_SETAFFINITY</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pf_mce_early">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PF_MCE_EARLY</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pf_mutex_tester">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PF_MUTEX_TESTER</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pf_freezer_skip">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PF_FREEZER_SKIP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pf_suspend_task">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PF_SUSPEND_TASK</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>GroupBox 2</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Executable</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_exe">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Command line</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_cmdline">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Working directory</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_cwd">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab_env">
   <attribute name="title">
    <string>Environment</string>
   </attribute>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidet_env">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>300</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Variable</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Value</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab_fds">
   <attribute name="title">
    <string>Opened files</string>
   </attribute>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QTableWidget" name="tableWidget_fds">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Descriptor</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Path</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab_perf">
   <attribute name="title">
    <string>Performance</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Fault</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>minflt</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_minflt">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>cminflt</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_cminflt">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>majflt</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_majflt">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>cmajflt</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_cmajflt">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string>Time</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>utime</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_utime">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>stime</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_stime">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>cutime</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_cutime">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>cstime</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_cstime">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>720</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>469</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
